<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcustomer</class>
 <widget class="QDialog" name="addcustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget {
    background-color: #1A2236;
    color: #FFFFFF;
    border: 1px solid #3A3F58;
    gridline-color: #3A3F58;
    font-family: &quot;Inter&quot;;
    font-size: 10pt;
    border-radius: 8px;
    selection-background-color: #0F5FFF;
    selection-color: #FFFFFF;
}

QHeaderView::section {
    background-color: #0F1A2C;
    color: #FFFFFF;
    border: 1px solid #3A3F58;
    padding: 4px;
    font-family: &quot;Montserrat SemiBold&quot;;
    font-size: 10pt;
}

QTableWidget::item:selected {
    background-color: #0F5FFF;
    color: #FFFFFF;
}
QGroupBox {
    border: 1.5px solid #BFC5D2;
    border-radius: 12px;
    margin-top: 25px;
    color: white;
    font-family: &quot;Segoe UI Semibold&quot;;
    font-size: 14pt;
}

QGroupBox::title {
    subcontrol-origin: margin;
    padding: 4px 10px;
}
QPushButton {
    background-color: #0F1A2C;
    color: #FFFFFF;
	border-color: rgb(0, 0, 0);
    border-radius: 10px;
    font-family: &quot;Montserrat SemiBold&quot;;
    font-size: 12pt;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #1A2942;
}

QPushButton:pressed {
    background-color: #09111A;
}
QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #C6D4E1;
    border-radius: 8px;
    padding: 6px;
    color: #0A1320;
    font-family: &quot;Inter&quot;;
    font-size: 11pt;
}

QLineEdit:focus {
    border: 1px solid #00B4FF;
    background-color: #F8FBFF;
}
QComboBox {
    background-color: #FFFFFF;
    border: 1px solid #C6D4E1;
    border-radius: 8px;
    padding: 6px;
    color: #0A1320;
    font-family: &quot;Inter&quot;;
    font-size: 11pt;
}

QComboBox:focus {
    border: 1px solid #00B4FF;
    background-color: #F8FBFF;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #C6D4E1;
}

QComboBox::down-arrow {
    
	image: url(:/new/prefix1/icon.png);
    width: 12px;
    height: 12px;
}</string>
  </property>
  <widget class="QGroupBox" name="gbox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Initialize the Account</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Account Holder Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="box">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Account Type :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0F1A2C;
    color: #FFFFFF;
border-color: rgb(0, 0, 0);
    border-radius: 8px;
    font-family: &quot;Segoe UI Semibold&quot;;
    font-size: 13pt;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0F1A2C;
    color: #FFFFFF;
border-color: rgb(0, 0, 0);
    border-radius: 8px;
    font-family: &quot;Segoe UI Semibold&quot;;
    font-size: 13pt;</string>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="box2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tunis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ariana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beja</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ben Arous</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bizerte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jendouba</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kef</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manouba</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nabeul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siliana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sousse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zaghouan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kairouan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kasserine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mahdia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monastir</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sidi Bouzid </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sfax</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gabes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kebili</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medenine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tataouine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tozeur</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Bank Branch :</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>331</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Register</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1,1,1,0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLineEdit" name="le1_2">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le2_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le3_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le4_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le5_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le7_2">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le6_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="l">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnback_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>271</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncreate_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>271</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/bg.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>gbox</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
